docs/guides/segment-migration-guide.md

```diff
@@ -0,0 +1,300 @@
+| ----------- | ----------- |
+| [Connections](https://segment.com/product/connections/) | [Sources](/data/sources/) & [Destinations](/data/destinations/) |
+| [Profiles](https://segment.com/product/profiles/)   | [Audiences](https://help.amplitude.com/hc/en-us/sections/360011146031-Amplitude-Audiences) |
+| [Protocols](https://segment.com/product/protocols/) | [Data Management](https://help.amplitude.com/hc/en-us/categories/5078631395227-Amplitude-CDP) |
+
+!!!info "Recommended Best Practice"
+    Follow a strict release process and [configure multiple environments](https://help.amplitude.com/hc/en-us/articles/5078848559259-Configure-and-manage-your-Amplitude-Data-settings#the-environments-tab). Validate changes within each environment before deploying.
+
+## Add a Source
+
+You can add a [new source](/data/sources/) in just a few clicks.
+
+1. From Data, click **Sources** in the Connections section.
+2. Click **Add Source**.
+3. Browse or search for the source you want to add.
+4. Follow the on-screen prompts.
+
+For detailed instructions, see the documentation for the [source](/data/sources/) you want to add.
+
+## Update SDK implementation
+
+Both Segment and Amplitude SDKs are meant to capture first party data by tracking user interactions. For the most part both work pretty similarly except some nuances around syntax. Here is high level mapping of concepts between Segment & Amplitude.
+
+| Segment   | Amplitude | Notes                                      |
+|-----------|-----------|--------------------------------------------|
+| write_key | api_key   | Unique key to validate source of the data. |
+| Workspace | Project   | [Projects](https://help.amplitude.com/hc/en-us/articles/360058073772-Create-and-manage-organizations-and-projects#create-a-project) allow you to organize your data.  |
+| User      | User      | User who is performing action.             |
+| Identify  | Identify  | [Identify](/analytics/what-is-amplitude/#user-properties-are-details-about-your-user) updates properties/attributes of the user.|
+| Track     | Event     | [Event](/analytics/apis/http-v2-api-quickstart/) in Amplitude tracks the action user is performing.|
+| Screen    | Event     | Create an Event to track Screen views.|
+| Page      | Event     | Create an Event to track Page views.|
+| Group     | Group     | [Group](/guides/accounts-instrumentation-guide/) is a collection of users. In Amplitude one user could belong to multiple groups. Each group can have properties/attributes that will be available to query/forward on actions performed by any user in the group.|
+| Plugins   | Plugins   | [Plugins](/data/ampli/plugin/) les you extend Amplitude by running a sequence of custom code on every event.|
+
+
+=== "Browser"
+
+    Documentation for [Browser Typescript SDK](/data/sdks/typescript-browser/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> Segment </td> <td> Amplitude </td>
+    </tr>
+    <tr>
+    <td> Identify </td>
+    <td> 
+    ```typescript
```

Comment:
This is a great workaround for adding code blocks to tables 🤯 

Might be worth documenting somewhere.

Reply:
Ha took me a bit to figure this out.